integreat_cms/cms/views/translations/translations_management_view.py

```diff
@@ -0,0 +1,95 @@
+import logging
+
+from django.shortcuts import redirect, render
+from django.core.exceptions import PermissionDenied
+from django.utils.decorators import method_decorator
+from django.views.generic import TemplateView
+from django.db import transaction
+
+from ...forms import TranslationsManagementForm
+
+from ...decorators import permission_required
+
+logger = logging.getLogger(__name__)
+
+
+@method_decorator(permission_required("cms.manage_translations"), name="dispatch")
+class TranslationsManagementView(TemplateView):
+    """
+    View for showing the machine translations management options
+    """
+
+    template_name = "translations/translations_management.html"
+
+    def get(self, request, *args, **kwargs):
+        r"""
+        Render translations management interface
+
+        :param request: The current request
+        :type request: ~django.http.HttpRequest
+
+        :param \*args: The supplied arguments
+        :type \*args: list
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+
+        :return: The rendered template response
+        :rtype: ~django.template.response.TemplateResponse
+        """
+
+        form = TranslationsManagementForm(instance=request.region)
+
+        return render(
+            request,
+            self.template_name,
+            {
+                **self.get_context_data(**kwargs),
+                "current_menu_item": "translations_management",
+                "form": form,
+            },
+        )
+
+    @transaction.atomic
+    def post(self, request, *args, **kwargs):
+        r"""
+        Submit :class:`~integreat_cms.cms.forms.translations.translations_management_form.TranslationsManagementForm` objects.
+
+        :param request: The current request
+        :type request: ~django.http.HttpRequest
+
+        :param \*args: The supplied arguments
+        :type \*args: list
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+
+        :raises ~django.core.exceptions.PermissionDenied: If user does not have the permission to edit the specific page
+
+        :return: The rendered template response
+        :rtype: ~django.template.response.TemplateResponse
+        """
+
+        if not request.user.has_perm("cms.manage_translations"):
+            raise PermissionDenied(
+                f"{request.user!r} does not have the permission to manage machine translation settings."
+            )
+
```

Comment:
```suggestion
```
Since you already required this permission via the
```
@method_decorator(permission_required("cms.manage_translations"), name="dispatch")
```
line for all http methods (`dispatch()` is called for both `get()` and `post()`, you don't need to check it here again.

Reply:
Oh and by the way, could you also add tests for the new view (both `get()` and `post()`) to the [view config](https://github.com/digitalfabrik/integreat-cms/blob/f973afc2eb7813ea96dd434c235c69cb3f4afd50/tests/cms/views/view_config.py#L31)?